policy/DecamMapper.yaml

```diff
@@ -167,20 +167,20 @@ calibrations:
     template: cpFLAT/%(calibDate)s/%(filter)s/FLAT-%(calibDate)s.fits[%(ccdnum)d]
     validRange: true
   defects:
-    columns:
-    - date
-    - ccdnum
-    filter: false
+    columns: date
     level: ccd
     obsTimeName: date
-    persistable: ImageU
-    python: lsst.afw.image.ImageU
-    refCols: visit
+    persistable: BaseCatalog
+    python: lsst.meas.algorithms.Defects
+    refCols:
+    - visit
     reference: raw_visit
     storage: FitsStorage
-    tables: defect
-    template: '%(path)s'
+    tables: defects
+    template: defects/%(calibDate)s/defects-%(calibDate)s-%(ccdnum)03d.fits
+    validEndName: validEnd
     validRange: true
+    validStartName: validStart
```

Comment:
None of the other calibs in the mapper specify validStartName or validEndName. Are these really required here?

Reply:
I think they will default to those strings.  I was just being explicit.  We can probably remove them, if you like.

Reply:
Makes sense, it's fine either way.